module/yzecoriolis.js

```diff
@@ -88,6 +79,22 @@ Hooks.once("init", async function () {
     label: "SheetClassItem",
   });
 
+  /**
+   * Set an initiative formula for the system
+   * @type {String}
+   */
+  if (game.settings.get("yzecoriolis", "Initiative")) {
```

Comment:
I think the logic is revered here. If true, the new formula should be used I think.